moderncviconsletters.sty

```diff
@@ -50,7 +50,7 @@
 \renewcommand*{\skypesocialsymbol}        {\textbf{sk}~}
 \renewcommand*{\orcidsocialsymbol}        {\textbf{orcid}~}
 \renewcommand*{\researchgatesocialsymbol} {\textbf{rg}~}
-\renewcommand*{\researchidsocialsymbol}   {\textbf{ri}~}
+\renewcommand*{\researchidersocialsymbol}   {\textbf{ri}~}
```

Comment:
Same here.

Reply:
done.

Reply:
Appears to be a typo: researcherid vs. researchider

Reply:
daaaaamn. thanks.